bin/node/cli/src/chain_spec.rs

```diff
@@ -284,7 +284,7 @@ pub fn testnet_genesis(
 			}).collect::<Vec<_>>(),
 		},
 		pallet_staking: StakingConfig {
-			validator_count: initial_authorities.len() as u32,
+			validator_count: (initial_authorities.len() * 2) as u32, // TODO: just for testing, revert at last.
```

Comment:
Should keep an eye on this and make sure all the testing-only changes get reverted before merge.

Reply:
I removed in this commit https://github.com/paritytech/substrate/pull/8912/commits/2f0b0a712f22b13c78c77bfa1a63341df09f6571